src/shader.frag

```diff
@@ -27,6 +27,13 @@ void main()
 
 
     float depth = 1.0 / gl_FragCoord.w; // the world space depth
+    
+    if (projection[2][3] != -1.0) {
+        // orthographic projection
+        depth_offset *= projection[2][2];
```

Comment:
What does this do? Can you explain in a comment?

Reply:
This divides depth_offset by (near - far)

Reply:
Yea I understand that but what is the effect of this division? What changes visually?